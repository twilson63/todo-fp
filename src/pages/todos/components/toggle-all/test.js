/* global test, expect */
import ToggleAll from './'
import React from 'react'
import { shallow } from 'enzyme'

test('ToggleAll Component renders correctly', done => {
  const props = {}
  const result = shallow(
    <ToggleAll {...props} />
  )

  expect(result).toMatchSnapshot()
  done()
})
