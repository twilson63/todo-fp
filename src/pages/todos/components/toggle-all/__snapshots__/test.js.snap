exports[`test ToggleAll Component renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <input
        className="toggle-all"
        onChange={[Function]}
        type="checkbox" />
    <label
        htmlFor="toggle-all">
        Mark all as complete
    </label>
</div>,
  "nodes": Array [
    <div>
      <input
            className="toggle-all"
            onChange={[Function]}
            type="checkbox" />
      <label
            htmlFor="toggle-all">
            Mark all as complete
      </label>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <ToggleAll />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": _class {
        "_reactInternalInstance": [Circular],
        "cachedHandlers": Object {},
        "context": Object {},
        "handlers": Object {
          "toggleAll": [Function],
        },
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <input
                    className="toggle-all"
                    onChange={[Function]}
                    type="checkbox" />
          <label
                    htmlFor="toggle-all">
                    Mark all as complete
          </label>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          <input
                    className="toggle-all"
                    onChange={[Function]}
                    type="checkbox" />
          <label
                    htmlFor="toggle-all">
                    Mark all as complete
          </label>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ToggleAll />,
}
`;
